ibm/mas_devops/roles/mirror_extras_prepare/vars/mongoce_6.0.12.yml

```diff
@@ -0,0 +1,26 @@
+---
+extra_images:
+  - name: mongodb/mongodb-kubernetes-operator
+    registry: quay.io
+    tag: 0.8.3
+    digest: sha256:9ef7a689b7f2789b436458a6d5f361ffcb4e182daa6c33d79687e87f268c7761 # 0.8.3-b20231215T000000Z
```

Comment:
is it confirmed these images do not disappear from day to day?

Reply:
These images are tagged with timestamp. So they won't disappear